src/modules/partition/core/BootLoaderModel.cpp

```diff
@@ -124,9 +125,9 @@ BootLoaderModel::updateInternal()
             appendRow( createBootLoaderItem( partitionText, PartitionInfo::mountPoint( partition ), true ) );
         }
 
-        // Create "don't install bootloader" item
-        appendRow( createBootLoaderItem( tr( "Do not install a boot loader" ), QString(), false ) );
     }
+	//Create "don't install bootloader" item
+	appendRow( createBootLoaderItem( tr( "Do not install a boot loader" ), QString(), false ) );
```

Comment:
And indentation looks funny here, too (it's not a big deal, the formatting tool will take care of it).